etl/src/main/java/org/pih/hivmigration/etl/sql/Migrator.java

```diff
@@ -150,6 +151,7 @@ public void main() {
                 migrate(new NutritionMigrator(), -1);
                 migrate(new TreatmentObsMigrator(), limit);
                 migrate(new DiagnosisMigrator(), limit);
+                migrate(new AccompagnateurMigrator(), limit);
```

Comment:
```suggestion
                migrate(new AccompagnateurMigrator(), -1);
```

This just makes it easy to identify as a migrator that doesn't do any loading.

Reply:
Thanks!